spec/git_reflow/2-lgtm/git_reflow_spec.rb

```diff
@@ -0,0 +1,668 @@
+require 'spec_helper'
```

Comment:
it's hard to tell what's new here, can you slim this down?  I don't mind the extra spec file for mutli-approvals here, but there's a lot of tests that don't seem to apply to the changes made.